### PR TITLE
new countries added

### PR DESCRIPTION
**Pull request by imran99744**
Added the NL (Netherlands) and CH (Switzerland) in the prayers-configs components.

### CODE REVIEW

File: src/countries/index.test.ts [hunk 1]
```diff
@@ -46,12 +46,22 @@ describe('country helpers', () => {
...
         expect(getConfigByISOName(opts.country)).toMatchSnapshot();
       },
-      Object.keys(prayersConfigs).map((country) => {
-        return {
-          name: country,
-          country,
-        };
-      })
+      [
+        ...Object.keys(prayersConfigs).map((country) => {
```
Comment (thehungrycoder) [inline src/countries/index.test.ts:50]: Why this change was needed? Didnt previous block capture all countries?

Comment (thehungrycoder) [reply src/countries/index.test.ts:50]: Check the snapshot. I think it now has dup snapshots

**Comment by imran99744** [comment]
@thehungrycoder I have added a new test case to cover the Netherlands and Switzerland in the `getConfigByISOName` function. This ensures that the function returns the correct configurations for these countries.

Could you please review the test case and let me know if it looks correct to you? Thank you.

File: src/countries/index.test.ts (hunk 1 above)
Comment (imran99744) [reply src/countries/index.test.ts:50]: Checking the snapshot. Should I add test inside the `describe('getConfigByISOName')` block to test the configurations for Netherlands and Switzerland?

Comment (thehungrycoder) [reply src/countries/index.test.ts:50]: please read the logic and understand it. 


if you didn't understand here is explanation: 
it already iterates on each country. so you don't need to add a country. but test will fail bcoz of snapshot mismatch. so all you need to review the snapshot differences and accept the update if they seem to be correct.

**Review by thehungrycoder — APPROVED** [review]
LGTM. Great work
